features/search-input-type.yml.dist

```diff
@@ -6,12 +6,12 @@ status:
   baseline_low_date: 2015-07-29
   baseline_high_date: 2018-01-29
   support:
-    chrome: "5"
+    chrome: ≤5
     chrome_android: "18"
     edge: "12"
-    firefox: "4"
+    firefox: ≤4
     firefox_android: "4"
-    safari: "5"
-    safari_ios: "4.2"
+    safari: ≤5
+    safari_ios: ≤4.2
```

Comment:
Yes, which is different from "it may have come earlier _or later_ than we report" which would be a natural outcome of "we're not sure we trust this data and haven't reviewed it carefully." But, when throwing together a bunch of things, the most recent version is the one we use, so there's good reason to expect that the error will usually be mostly in the direction of claiming features are newer than they are.